### PR TITLE
Matlab interface

### PR DESCRIPTION
**Pull request by FreyJo**
- More clear installation instructions for the matlab interface
- Bug fix for matlab interface

### CODE REVIEW

File: interfaces/python/hpipm_python/hpipm_python/wrapper/hpipm_ocp_qp.py [hunk 1]
```diff
@@ -167,11 +167,19 @@ def __init__(self, dim):
 
 	
 	def set(self, field, value, idx=None):
+		# cast to np array
 		if type(value) is not np.ndarray:
-			if type(value) is int:
+			if (type(value) is int) or (type(value) is float):
```
Comment (giaf) [inline interfaces/python/hpipm_python/hpipm_python/wrapper/hpipm_ocp_qp.py]: why setting a dim as float?

File: examples/matlab/getting_started.m [hunk 2]
```diff
@@ -1,7 +1,7 @@
 % pyversion /usr/bin/python3
 % py.sys.path
 
-clear all
+clearvars
```
Comment (giaf) [inline examples/matlab/getting_started.m]: why this change?

File: interfaces/python/hpipm_python/hpipm_python/wrapper/hpipm_ocp_qp.py (hunk 1 above)
Comment (FreyJo) [reply interfaces/python/hpipm_python/hpipm_python/wrapper/hpipm_ocp_qp.py]: As far as I understand, this is not only a setter for dims but also for bound values, thus floats make sense.

File: examples/matlab/getting_started.m (hunk 2 above)
Comment (FreyJo) [reply examples/matlab/getting_started.m]: Recent Matlab versions underline `clear all` and suggest to use `clearvars` instead.
It is supposed to be faster, I think.
